src/components/PlayerItem.vue

```diff
@@ -32,7 +32,8 @@ const handlePlayerSaidGabo = (playerName: string) => {
     </div>
     <div class="bottom">
       <div class="counter">
-        <button @click="onDecrementPoints(player.name)">–</button>
+        <button v-if="points >= 1" @click="onDecrementPoints(player.name)">–</button>
+        <div v-if="points < 1" class="counter__blank-space"></div>
```

Comment:
don't create a new node. instead use the visibility: hidden. set conditional class for the button